qtl/tree.h

```diff
@@ -457,15 +457,15 @@ inline static const std::map<std::string,std::function<operand_t(const std::vect
 #define Xright2(n,o)  Xin2(n,o)
 #define Xleft_(n,o)  T(n,o, NOTRACE( std::cerr << __PRETTY_FUNCTION__  <<'\n'; )  \
   if( v.size()==0 ){ \
-    return n ## _identity; \
+    return operand_t(n ## _identity);		\
```

Comment:
Maybe the cast will be unnecessary after resolving the overloaded ‘scalar(int)
[](url) (is there a way to make a link from this comment to the comment above?)